lib/loadGltfUris.js

```diff
@@ -0,0 +1,52 @@
+'use strict';
+var fs = require('fs');
+var Cesium = require('cesium');
+var defined = Cesium.defined;
+
+module.exports = loadGltfUris;
+
+function loadGltfUris(path, gltf, callback) {
+    var shaders = gltf.shaders;
+    // process.stdout.write('START\n');
+    // console.log(gltf);
+    if (defined(shaders)) {
+        for (var id in shaders) {
+            if (shaders.hasOwnProperty(id)) {
+                var shader = shaders[id];
+                // process.stdout.write('SHADER: ' + shader + '\n');
+                // var extras = shader.extras;
+                // if (!defined(extras)) {
+                //     shader.extras = {};
+                // }
+                // extras = shader.extras;
+                shader.extras = shader.extras || {};
```

Comment:
Is this line okay in terms of style, or should I use the commented version instead?